Oracle_CQRS/.gitattributes

```diff
@@ -0,0 +1,2 @@
+/mvnw text eol=lf
```

Comment:
Any git files, can you add to the root project git files unless there is a compelling reason to have them separate?